DemoPage/examples/DateSelect/index.js

```diff
@@ -1,20 +1,21 @@
-"use strict";
+'use strict';
 
 var React             = require('react');
 var DateSelect        = require('../../../forms/DateSelect');
-var DateSelectWrapper = require('../../../forms/DateSelect/Wrapper');
+var DateSelectWrapper = require('../../../forms/DateSelect/DateSelectWrapper');
```

Comment:
Consistency. Component files, displayNames, and folder names should all match. It also shouldn't be nested under DateSelect, so I might fix that too. One component per folder, always.